singer/utils.py

```diff
@@ -134,6 +134,7 @@ def parse_args(required_config_keys):
     -d,--discover   Run in discover mode
     -p,--properties Properties file: DEPRECATED, please use --catalog instead
     --catalog       Catalog file
+    -dev, --dev     Runs the tap in dev mode
```

Comment:
Is there really any value to having `-dev`? If we want a short flag, can we pick some other letter?

In some circles, this would read as "I'm passing `-d`, `-e`, and `-v` and POSIX convention says I can chain them together". And this is not something I want us to introduce to this CLI

Reply:
Oh yeah, I missed that one. I really to like the convention of having the short flag be only one letter. I'd even be good with `-D`.

Reply:
If there is no dependancy on `-dev` switch anywhere else then I am also in favour of removing it. Should I remove it or should we replace it alternate short letter like `-e`?